com.unity.netcode.gameobjects/Editor/Configuration/NetworkPrefabProcessor.cs

```diff
@@ -0,0 +1,160 @@
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+namespace Unity.Netcode.Editor.Configuration
+{
+    /// <summary>
+    /// Updates the default <see cref="NetworkPrefabsList"/> instance when prefabs are updated (created, moved, deleted) in the project.
+    /// </summary>
+    public class NetworkPrefabProcessor : AssetPostprocessor
+    {
+        private static string s_DefaultNetworkPrefabsPath = "Assets/DefaultNetworkPrefabs.asset";
+        public static string DefaultNetworkPrefabsPath
+        {
+            get
+            {
+                return s_DefaultNetworkPrefabsPath;
+            }
+            internal set
+            {
+                s_DefaultNetworkPrefabsPath = value;
+                // Force a recache of the prefab list
+                s_PrefabsList = null;
+            }
+        }
+        private static NetworkPrefabsList s_PrefabsList;
+
+        // Unfortunately this method is required by the asset pipeline to be static
```

Comment:
That comment came from @JesseOlmer so I won't speak for him as to the intention behind it.

Reply:
```suggestion
```

This was mostly a note to myself during experimentation... It's unfortunate because static things are harder to mock, override, disable, etc. I didn't actually write ANY tests for any of this code so not sure how much this being static actually made things harder ;)

Reply:
Had a feeling that was your thought process here... and yeah, it made testing harder... tests are having to set static variables and make sure they unset them at the end because if they don't, the actual editor settings get changed... it's yucky.

Reply:
Yeah...it is what it is:
https://docs.unity3d.com/2022.2/Documentation/ScriptReference/AssetPostprocessor.OnPostprocessAllAssets.html
One of those scenarios where creating an internal set of callbacks might help (?) with testing (and be valid due to it being static and all).